src/PaloAltoNetworks/PaloAltoNetworks.Autorest/test/AzPaloAltoNetworks.Tests.ps1

```diff
@@ -19,156 +19,156 @@ Describe 'AzPaloAltoNetworks' {
         {
             $config = New-AzPaloAltoNetworksLocalRulestack -Name $env.LocalRulestackName -ResourceGroupName $env.resourceGroup -Location $env.location -Description "testing powershell" -DefaultMode 'NONE'
             $config.Name | Should -Be $env.LocalRulestackName
-        } | Should -Not -Throw
```

Comment:
Is special reason that you remove `Should -Not -Throw` Here.

Reply:
The build was failing due to missing exception

https://dev.azure.com/azclitools/public/_build/results?buildId=110806&view=logs&j=b17395f6-68a3-5682-0476-d3f6f1043109&t=e50c9eb1-7f19-5aa6-670b-cc46e51b352e

Reply:
> Is special reason that you remove `Should -Not -Throw` Here.

@osamasid would please revert the change to this file?

Reply:
I reverted the changes and the CI checks are passing now